README.md

```diff
@@ -1,109 +1,31 @@
-  - **info:** Files containing information that the implementer might find useful. These may be files on language-specific elements of programming concepts or notes on the idiomatic way to use tooling. For now, maintainers are encouraged to organise this directory however they feel best, and we may formalise a structure later.
-  - **concept-exercises:** The WIP concept exercises for that language.
-  - **config.json** The WIP config.json for a track, with the practice exercises removed to avoid noise.
-
-- **tooling** This directory contains documents on tooling that is common across languages (either due to a shared ecosystem - such as JS/TS/Clojurescript - or a shared platform - such as .net or JVM).
-- **types:** This directory contains files discussing programming types. These documents should provide guidance, limitations, and links for maintainers who need to expound those ideas on their tracks.
-
-## How to get started
-
-### Filling out concepts for your language
-
-To know which Concept Exercises should exist, a list of the language's concepts should first be compiled. There are various resources to use when compiling this list: books, (official) documentation, your own experiences and the concepts listed in this repository.
-
-As many languages have the same concepts (but possibly implemented differently), it can greatly help to look at what other languages have already done in this regard. See the [C# README.md](./languages/csharp/README.md) for an example.
-
-It can also be helpful to group related concepts. For example, the Classes, Polymorphism and Inheritance concepts are all object-oriented concepts, which can then be grouped as "Object-oriented concepts".
-
-Filling out the concepts will be iterative, it is hard to get this right immediately. While creating concepts exercises, you'll probably find missing concepts or perhaps want to split out concepts.
-
-### Writing your first concept exercise
-
-Once a list of concepts has been compiled, these then need to be translated into Concept Exercises. Writing your first concept exercise begins with choosing a concept to write an exercise for. Good first concept candidates are concepts that involve working with basic types, like strings and numbers.
-
-Having chosen a concept, look for the concept's description in this repository. This file should have references to all tracks that have implemented the exercise. You can use these existing implementations as a starting point for your first concept exercise, which you can then tailor to your specific language.
+- [What are concept exercise are and how they are structured?](./docs/concept-exercises.md)
+- [How is this repository structured?](./docs/repository-structure.md)
```

Comment:
The `repository-structure.md` file is in the `docs/maintainers` directory. Looking at the sidebar definitions, I'm starting to think it should be in `docs`, and that this link is actually correct, but the file's location isn't.